src/components/td/td-styles.jsx

```diff
@@ -20,12 +20,61 @@ export default theme => {
       minWidth: 20,
       paddingTop: 8,
       paddingBottom: 8,
+      display: 'block',
```

Comment:
A `<td>` should likely never be set as `block` as they are normally `table-cell`. Perhaps in mobile, but we'd likely not use a table then...